codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -350,7 +349,7 @@ private static void writeChildNode(FlowState state) {
         FlowState(Http2Stream stream, int initialWindowSize) {
```

Comment:
I think the package private access *maybe* for 2 reasons:

1. Used in tests?
2. Avoid synthetic method generation?

So I'm not sure we need the package private access but if we could make more restrictive that is good too.